Azure/azure-cli
# {Storage} Use SDK definition of DefaultSharePermission

**Description**

Previously, CLI overwrites SDK's `DefaultSharePermission` definition as there's one permission server doesn't support.
SDK will fix this by https://github.com/Azure/azure-rest-api-specs/pull/15627/commits/9ea84fe80f7f780f3861bb82af8516b3ade9a7db
So it's time to use SDK definition


**Testing Guide**

`az storage account create/update`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
